src/Sdk/WebApi/WebApi/LaunchContracts.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace GitHub.Services.Launch.Contracts
+{
+    [DataContract]
+    public class ActionReferenceRequest
+    {
+        [DataMember(EmitDefaultValue = false, Name = "action")]
+        public string Action { get; set; }
+
+        [DataMember(EmitDefaultValue = false, Name = "version")]
+        public string Version { get; set; }
+
+        [DataMember(EmitDefaultValue = false, Name = "path")]
+        public string Path { get; set; }
+    }
+
+    [DataContract]
+    public class ActionReferenceRequestList
```

Comment:
why do we need an object to wrap around a list?
Can we just send the List as json to service?